airbyte-webapp/src/components/ServiceForm/components/FormSection.tsx

```diff
@@ -129,12 +129,37 @@ const ArraySection: React.FC<{
   );
 };
 
+function OrderComparator(a: FormBlock, b: FormBlock): number {
```

Comment:
we should fall back on sorting alphabetically if we don't have an order field. i'm not seeing that logic here. does that still need to be implemented it or am i being blind?

Reply:
Yeah. We do check on line 156 and do not sort at all if no order present.

Reply:
great!